### PR TITLE
Add image on contact category list

### PR DESCRIPTION
**Pull request by drmenzelit**
com_contact has the possibility to add photos to each contact. The photo can be showed on the view of a single contact, but not in the list of contacts inside a category.
I added two new fields under the List Layouts options to be able to display the contact photo in the list of contacts:
screenshot 141

That is the view (I added Bootstrap 2 and 3 classes to make it compatible with the two versions of the framework):
screenshot 142

I also added a definition into the CSS file of Protostar to correct the display of the rows.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_contact.ini [hunk 1]
```diff
@@ -56,6 +56,12 @@ COM_CONTACT_FIELD_CONFIG_FAX_DESC="Show or hide a Fax column in the list of Cont
 COM_CONTACT_FIELD_CONFIG_FAX_LABEL="Fax"
 COM_CONTACT_FIELD_CONFIG_INDIVIDUAL_CONTACT_DESC="These settings apply for single Contact unless they are changed for a specific menu item or Contact."
 COM_CONTACT_FIELD_CONFIG_INDIVIDUAL_CONTACT_DISPLAY="Contact"
+COM_CONTACT_FIELD_SHOW_LIST_COLS_DESC="Select the width of the columns for image, description and extras (Bootstrap)"
+COM_CONTACT_FIELD_SHOW_LIST_COLS_LABEL="Columns width (Bootstrap)"
+COM_CONTACT_FIELD_SHOW_LIST_COLS_1="2,7,3"
```
Comment (andrepereiradasilva) [inline administrator/language/en-GB/en-GB.com_contact.ini]: I think you should check this https://github.com/joomla/joomla-cms/pull/8300 reggarding lang vars ending with "_1"

**Comment by infograf768** [comment]
Have you tested the CSS with a RTL language?

**Comment by drmenzelit** [comment]
What do you mean infograf768? I used the standard Protostar to test. I think if someone wants to change the order of the columns to show from right to left, then he/she has to create an override of the view.

**Comment by infograf768** [comment]
was just wondering about
```
.contact-category .row {
    margin-left: 0;
}
```
but, after testing, it looks like it is OK both in RTL and LTR

**Comment by drmenzelit** [comment]
I added this line to the CSS because in Protostar "row" is defined with margin-left: -20px

**Comment by waader** [comment]
I have tested this item :white_check_mark: successfully on 72746287da318861b231c00ab882603b1c5bda01<br /><br />Works fine, thanks! Shouldn´t this also be added to Beez?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8580">issues.joomla.org/joomla-cms/8580</a>.</sub>

**Comment by drmenzelit** [comment]
It can also be added to the override of Beez...

**Comment by kleinhelmi** [comment]
I have tested this item :white_check_mark: successfully on 72746287da318861b231c00ab882603b1c5bda01<br /><br />tested with Joomla 3.5.0-beta @ Apache/2.4.10 (Win32) OpenSSL/1.0.1i PHP/5.5.15<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8580">issues.joomla.org/joomla-cms/8580</a>.</sub>

File: administrator/language/en-GB/en-GB.com_contact.ini [hunk 2]
```diff
@@ -56,6 +56,12 @@ COM_CONTACT_FIELD_CONFIG_FAX_DESC="Show or hide a Fax column in the list of Cont
 COM_CONTACT_FIELD_CONFIG_FAX_LABEL="Fax"
 COM_CONTACT_FIELD_CONFIG_INDIVIDUAL_CONTACT_DESC="These settings apply for single Contact unless they are changed for a specific menu item or Contact."
 COM_CONTACT_FIELD_CONFIG_INDIVIDUAL_CONTACT_DISPLAY="Contact"
+COM_CONTACT_FIELD_SHOW_LIST_COLS_DESC="Select the width of the columns for image, description and extras (Bootstrap)"
+COM_CONTACT_FIELD_SHOW_LIST_COLS_LABEL="Columns width (Bootstrap)"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_contact.ini]: This should be Column not columns

File: administrator/language/en-GB/en-GB.com_contact.ini [hunk 3]
```diff
@@ -56,6 +56,12 @@ COM_CONTACT_FIELD_CONFIG_FAX_DESC="Show or hide a Fax column in the list of Cont
...
+COM_CONTACT_FIELD_SHOW_LIST_COLS_SMALL="2,7,3"
+COM_CONTACT_FIELD_SHOW_LIST_COLS_WIDE="3,6,3"
+COM_CONTACT_FIELD_CONFIG_SHOW_IMAGE_LABEL="Image" 
+COM_CONTACT_FIELD_CONFIG_SHOW_IMAGE_DESC="Show or hide a Image column in the list of Contacts."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_contact.ini]: This should be 
an Image

**Comment by mikeveeckmans** [comment]
I have tested this item :white_check_mark: successfully on 2a8f4f4290d459b9758eb714368054d464ca37e4<br /><br />TEST OK<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8580">issues.joomla.org/joomla-cms/8580</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8580">issues.joomla.org/joomla-cms/8580</a>.</sub>

**Comment by wilsonge** [comment]
OK I'm going to make an executive decision here. Can we remove the param for setting column widths please? It's unnecessary - we have far too many params as it is - template overrides are easy to create for that sort of thing

**Comment by drmenzelit** [comment]
@wilsonge I removed the parameter for column width

**Comment by wilsonge** [comment]
Thanks! Now can you add back the installation directory please :)

**Comment by mikeveeckmans** [comment]
ready for test ? <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8580">issues.joomla.org/joomla-cms/8580</a>.</sub>

**Comment by brianteeman** [comment]
Closed please retest #9906<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8580">issues.joomla.org/joomla-cms/8580</a>.</sub>

**Pull request closed without merging**